libs/langchain/langchain/callbacks/tracers/evaluation.py

```diff
@@ -9,7 +9,7 @@
 import langsmith
 from langsmith import schemas as langsmith_schemas
 
-from langchain.callbacks.manager import tracing_v2_enabled
+from langchain import callbacks
```

Comment:
I think in general we might want to move away from importing things internally from init files?